SEGSupport/EncodeSEG.cxx

```diff
@@ -38,12 +38,12 @@
 #include <itkBinaryThresholdImageFilter.h>
 
 // UIDs
-#include "../Common/QIICRUIDs.h"
-#include "../Common/SegmentAttributes.h"
+#include "QIICRUIDs.h"
+#include "SegmentAttributes.h"
 //#include "../Common/conditionCheckMacros.h"
 
 // versioning
-#include "../Iowa2DICOMVersionConfigure.h"
+#include "../vtkReportingVersionConfigure.h"
```

Comment:
The relative path here seems more readable and explicit - otherwise the info is hidden away and it's not obvious where to look for this (already somewhat magical) file.